hail/src/main/scala/is/hail/expr/types/physical/ComplexPType.scala

```diff
@@ -14,4 +14,4 @@ abstract class ComplexPType extends PType {
   override def fundamentalType: PType = representation.fundamentalType
 
   override def containsPointers: Boolean = representation.containsPointers
-}
+}
```

Comment:
prefer to have whitespace at the end of file -- is your editor doing this automatically?

Reply:
It just doesn't add whitespace. I'll make sure IntelliJ is pointed to the right format config, maybe that is the issue.